docs/openapi.json

```diff
@@ -196,30 +196,12 @@
                         "content": {
                             "application/json": {
                                 "schema": {
-                                    "$ref": "#/components/schemas/HealthResponse"
+                                    "$ref": "#/components/schemas/ReadinessResponse"
                                 }
                             }
                         }
                     }
                 }
-            },
```

Comment:
are not HEAD HTTP methods used regularly, especially for liveness and readiness probes? @JoaoFula ?

Reply:
yeah, I believe so. Don't normally use it myself but that's their purpose.